drivers/sensor/wsen_hids/wsen_hids_trigger.c

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (c) 2022 Würth Elektronik eiSos GmbH & Co. KG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT we_wsen_hids
+
+#include <zephyr/logging/log.h>
+
+#include "wsen_hids.h"
+
+LOG_MODULE_DECLARE(WSEN_HIDS, CONFIG_SENSOR_LOG_LEVEL);
+
+static inline void hids_setup_drdy_interrupt(const struct device *dev, bool enable)
+{
+	const struct hids_config *cfg = dev->config;
+	unsigned int flags = enable ? GPIO_INT_EDGE_TO_ACTIVE : GPIO_INT_DISABLE;
```

Comment:
type should be `gpio_flags_t`?

https://docs.zephyrproject.org/apidoc/latest/group__gpio__interface.html#ga5f5cb5e7dae6d58e072bb450af029d2e